src/Serilog.Sinks.NewRelic/Serilog.Sinks.NewRelic.nuspec

```diff
@@ -1,33 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Why is nuspec removed? Shouldn't it be rather be updated accordingly?

Reply:
With the update to using the newer csproj file format, a lot of the properties defined in the .nuspec file can go into the csproj file instead, and this is what [Microsoft recommends](https://docs.microsoft.com/en-us/nuget/reference/nuspec#project-type-compatibility):

> If you are creating a package using dotnet.exe pack or msbuild pack target, we recommend that you include all the properties that are usually in the .nuspec file in the project file instead. 

I think that .nuspec are still supported if you'd prefer that we keep it, though.

Reply:
I see. We are still on .NET Framework 4.6 in the project which uses (or, rather, used) this library, so I haven't looked much into the finer points of the Core.

Reply:
I've made some local changes (not yet pushed) to the csproj file (migrated over author and license information) as you can see in the screenshot below:

![image](https://user-images.githubusercontent.com/1588951/77526707-ec733b00-6e82-11ea-9d25-fd794fa81bd9.png)

One thing though - it seems to include the icon when viewed through Package Explorer, but that icon isn't appearing in the NuGet Package Manager. I'm not sure if this is just a caching issue that will fix itself.

Reply:
The file structure is correct, so must be caching. And do add your name to the authors list.